scripts/init-submodules-no-riscv-tools.sh

```diff
@@ -59,13 +40,7 @@ git config --unset submodule.generators/sha3.update
 # Shallow clone by default (recursive clone would get linux)
 git submodule update --init generators/sha3
 
-if [ $NO_FIRESIM = false ]; then
-  echo "initializing firesim"
-  # Renable firesim and init only the required submodules to provide
-  # all required scala deps, without doing a full build-setup
-  git config --unset submodule.sims/firesim.update
-  git submodule update --init sims/firesim
-  git -C sims/firesim submodule update --init sim/midas
-  git -C sims/firesim submodule update --init --recursive sw/firesim-software
-  git config submodule.sims/firesim.update none
```

Comment:
I think my preference would be re-disable the the firesim submodule if that's alright with you Albert.

Reply:
+1

Reply:
Done.